dbt/adapters/glue/gluedbapi/connection.py

```diff
@@ -369,7 +369,6 @@ class SqlWrapper2:
     dfs = {}
     @classmethod
     def execute(cls,sql,output=True):
-        sql = sql.replace('"', '')
```

Comment:
@moomindani What was the purpose of this sql change ? It is currently breaking code running with double quotes

Reply:
This was my bad, and I was preparing for fixing this in the next PR. Thanks for your PR.

Reply:
Thank you for your response! no worries